gitbook-docs/subscription_protocol.md

```diff
@@ -1,194 +1,198 @@
+   values.
  * `fixed` means simply send the last known values every `period`.
-* `minPeriod=[millisecs]` becomes the fastest message transmission rate allowed, e.g. every `minPeriod/1000` seconds. This is only relevant for policy='instant' to avoid swamping the client or network.
+* `minPeriod=[millisecs]` becomes the fastest message transmission rate allowed, e.g. every `minPeriod/1000` seconds.
+    This is only relevant for policy='instant' to avoid swamping the client or network.
+
+You can subscribe to multiple data keys multiple times, from multiple apps or devices. Each app or device simply
+subscribes to the data it requires, and the server and/or client implementation may combine subscriptions to avoid
+duplication as it prefers on a per connection basis. At the same time it is good practice to open the minimum
+connections necessary, for instance one WebSocket connection shared between an instrument panel with many gauges,
+rather then one WebSocket connection per gauge.
 
-You can subscribe to multiple data keys multiple times, from multiple apps or devices. Each app or device simply subscribes to the data it requires, and the server and/or client implementation may combine subscriptions to avoid duplication as it prefers on a per connection basis. At the same time it is good practice to open the minimum connections necessary, for instance one websocket connection shared bewteen an instrument panel with many gauges, rather then one websocket connection per gauge.
+## Multiple value handling in subscriptions
 
-#### Multiple value handling in subscriptions
+A subscription to a key is for all the updates to that key. If there are multiple sources generating data for that key
+the client will get all their updates.
 
-A subscription to a key is for all the updates to that key. If there are multiple sources generating data for that key the client will get all their updates.
+If a client wants only the values of a single source it should subscribe to a path that includes the full path under
+`values` including the source reference key of the source. The source reference should be enclosed in square brackets:
+`navigation.speedThroughWater.values[n2kFromFile.43]`. The client can retrieve the relevant data via REST API.
 
-If a client wants only the values of a single source it should subscribe to a path that includes the full path under `values` including the source reference key of the source. The source reference should be enclosed in square brackets:  `navigation.speedThroughWater.values[n2kFromFile.43]`. The client can retrieve the relevant data via REST API.
+## Single use, or intermittent data
 
-#### Single use, or intermittent data
+When data is required once only, or upon request the `subscribe/unsubscribe` method should not be used. If the client
+is http capable the REST API is a good choice, or use `get/list/put` messages over WebSockets or TCP.
```

Comment:
Node server does not support GET/LIST/PUT over ws. Maybe I've forgotten something, but I have yet to see good justification for inclusion of it in the core protocol.

I suggest we remove these from the spec.